src/app/components/UserMenu/index.tsx

```diff
@@ -35,6 +35,10 @@ export default function UserMenu() {
   async function lock() {
     try {
       auth.lock(() => {
+        const allTabs = chrome.extension.getViews({ type: "tab" });
```

Comment:
It's best to use the `browser` polyfill as we do this in Firefox as well.
You can shift this to https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/state.ts#L119 so that you won't have to import `browser` again.
And the `window.close()` would stay as the callback (to `auth.lock` fn) to close the popup

Reply:
I tried adding the same code here https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/state.ts#L119 as suggested, but this is not working. I tried adding `console.log` statements in order to find out whether this function is even getting called or not when we click on lock option on the popup, but the logging were not presents which basically confirms this function is not getting called. So can I just replace chrome with browser polyfill in this file only?

Reply:
You are checking these logs, right? (you should be)
<img width="414" alt="Screenshot 2023-02-07 at 1 09 17 PM" src="https://user-images.githubusercontent.com/64399555/217181826-760b72f6-949e-4ac4-82eb-cc968aa43545.png">

Reply:
This works for me 💁‍♂️ 
```
lock: async () => {
    const allTabs = browser.extension.getViews({ type: "tab" });
    for (const tab of allTabs) {
      tab.close();
    }
    const connector = get().connector;
    if (connector) {
      await connector.unload();
    }
    set({ password: null, connector: null, account: null, nostr: null });
  }

Reply:
Yes, it is working. Actually I need to restart the server in order to see the changes in effect. Sorry for the confusion.